statics/js/components/rule-detail.js

```diff
@@ -580,65 +579,80 @@ Vue.component('groups-detail', {
   },
 });
 
+
 /** Vue component showing the rules associated to a bridge */
 Vue.component('rule-detail', {
+
+  mixins: [apiMixin],
+
   template: '\
-<div class="rules-detail flow-ops-panel" v-if="Object.keys(layout.structured).length > 0">\
+    <div class="rules-detail flow-ops-panel" v-if="Object.keys(layout.structured).length > 0">\
```

Comment:
i don't know what changed but the background color is not the same as previously. Can you use the same "style" as flow  filtering for the filtering button/footer.